static/js/components/CreateCommentForm.js

```diff
@@ -47,10 +47,19 @@ const commentForm = (
   onUpdate: (e: any) => void,
   cancelReply: () => void,
   isComment: boolean,
-  disabled: boolean
+  disabled: boolean,
+  autoFocus: boolean
 ) =>
   <div className="reply-form">
-    <form onSubmit={onSubmit} className="form">
+    <form
+      onSubmit={onSubmit}
+      className="form"
+      onKeyDown={e => {
+        if (e.key === "Enter" && e.ctrlKey) {
+          onSubmit(e)
```

Comment:
Can you add a check that this doesn't happen if the button is disabled?

Reply:
I added that functionality and added a few tests for it as well